course/templatetags/base.py

```diff
@@ -44,6 +45,9 @@ def site_alert():
     if message:
         return mark_safe('<div class="alert alert-danger">{}</div>'
                          .format(pick_localized(message)))
+    if not grader_stable():
```

Comment:
If exercise tests are broken, wouldn't grader still be able to reply with some error (maybe after timeout), instead of just silently dropping the response? So not getting any response would be an some error in grader itself (or in the network, etc.)? But we could of course make this also course specific.

In other context we have also discussed making the sitewide alert adjustable during runtime instead of through settings.py. Maybe at the same time we could implement system for course-specific alerts, aside with run-time sitewide alerts. Then also this kind of "automatic" alerts could be implemented using this new approach. But if we would do something like this, I guess it would be a separate PR and commit. So not doing anything about this yet, pending follow-up discussion.

Reply:
So if the grader responds and sets the submission to the error status in A+, then this PR won't consider the grader unstable and there won't be any warnings in the site? That's an important distinction, indeed.

Submission status in the grader feedback meta attributes:
https://github.com/apluslms/mooc-grader/blob/1f5b16365930f5b18de4c39e427ce2be7e0884d6/access/templates/access/exercise_head.html#L13